### PR TITLE
Fix or suppress modernizer errors

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     </parent>
 
     <properties>
-        <platform.check.fail-modernizer>false</platform.check.fail-modernizer>
```
Comment (gwittel) [inline pom.xml:18]: Minor: We should document this behavior change in the changelog.

Comment (johngmyers) [reply pom.xml:18]: That is in the platform pom, not the library pom, so only affects Platform proper.

Comment (gwittel) [reply pom.xml:18]: Ah oops, missed that.

**Review by gwittel — APPROVED** [review]
LGTM

**Pull request merged**